runtime/gc_base/GCExtensions.hpp

```diff
@@ -219,7 +219,7 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	static MM_GCExtensions* newInstance(MM_EnvironmentBase* env);
 	virtual void kill(MM_EnvironmentBase* env);
 
-	void computeDefaultMaxHeapForJava(bool enableOriginalJDK8HeapSizeCompatibilityOption);
```

Comment:
let's be consistent and change all existing UDATA/IDATA in GCExtensions.hpp/.cpp uintptr_t/intptr_t

Reply:
Replaced in the newest commit, thanks.